src/server/replica.cc

```diff
@@ -154,6 +155,13 @@ void Replica::Stop() {
   for (auto& flow : shard_flows_) {
     flow.reset();
   }
+  if (last_journal_LSNs_.has_value()) {
+    LastMasterSyncData data;
+    data.id = master_context_.master_repl_id;
+    data.last_journal_LSNs = last_journal_LSNs_.value();
+    return data;
```

Comment:
return
```suggestion
  return {master_context_.master_repl_id, last_journal_LSNs_.value()};
```

Reply:
We can do this instead:

```
return LastMasterSyncData{master_context_.master_repl_id, last_journal_LSNs_.value()};
```

And it will RVO it into the optional :)

p.s. not strongly opinionated about this -- whatever works